src/test/java/com/mindee/product/invoice/InvoiceV4Test.java

```diff
@@ -42,30 +41,29 @@ void whenEmptyDeserialized_mustHaveValidProperties() throws IOException {
     Assertions.assertNull(docPrediction.getDueDateField().getValue());
     Assertions.assertNull(docPrediction.getTotalNet().getValue());
     Assertions.assertNull(docPrediction.getTotalAmount().getValue());
+    Assertions.assertNull(docPrediction.getTotalTax().getValue());
     Assertions.assertTrue(docPrediction.getTaxes().isEmpty());
     Assertions.assertTrue(docPrediction.getSupplierPaymentDetails().isEmpty());
     Assertions.assertNull(docPrediction.getSupplierName().getValue());
-    Assertions.assertNull(docPrediction.getTotalTax().getValue());
     Assertions.assertTrue(docPrediction.getSupplierCompanyRegistrations().isEmpty());
     Assertions.assertNull(docPrediction.getSupplierAddress().getValue());
     Assertions.assertNull(docPrediction.getCustomerName().getValue());
     Assertions.assertTrue(docPrediction.getCustomerCompanyRegistrations().isEmpty());
     Assertions.assertNull(docPrediction.getCustomerAddress().getValue());
-    Assertions.assertNotNull(docPrediction.getDocumentType().getValue());
+    Assertions.assertNull(docPrediction.getShippingAddress().getValue());
+    Assertions.assertNull(docPrediction.getBillingAddress().getValue());
+    Assertions.assertInstanceOf(ClassificationField.class, docPrediction.getDocumentType());
     Assertions.assertTrue(docPrediction.getLineItems().isEmpty());
   }
 
   @Test
   void whenCompleteDeserialized_mustHaveValidDocumentSummary() throws IOException {
     PredictResponse<InvoiceV4> response = getPrediction("complete");
     Document<InvoiceV4> doc = response.getDocument();
-    String[] actualLines = doc.toString().split(System.lineSeparator());
-    List<String> expectedLines = Files.readAllLines(
-      Paths.get("src/test/resources/products/invoices/response_v4/summary_full.rst")
+    ProductTestHelper.assertStringEqualsFile(
+        doc.toString(),
+        "src/test/resources/products/invoices/response_v4/summary_full.rst"
     );
```

Comment:
Rly wish there was an equivalent of this in some other libs :(

Reply:
it's not exactly rocket surgery ;-)
 
https://github.com/mindee/mindee-api-java/blob/main/src/test/java/com/mindee/product/ProductTestHelper.java#L11